spec/features/admin_adds_video_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'spec_helper'
+
+feature 'admin adds video' do
+  
+  scenario "admin successfully adds a new video" do
+    
+    admin_user = object_generator(:admin)
+    
+    sign_in_user(admin_user)
+    add_video
+    
+    sign_out
+    sign_in_user
+    
+    visit video_path(Video.first)
+    expect(page).to have_selector("img[src='https://knoxjeffrey-myflix-development.s3.amazonaws.com/uploads/monk_large.jpg']")
```

Comment:
I think you could abstract these 2 to something like `validates_uploaded_image` - to be aligned with upper abstracted code.